examples/demo-apps/apple_ios/ExecuTorchDemo/ExecuTorchDemo.xcodeproj/project.pbxproj

```diff
@@ -59,6 +59,13 @@
 			remoteGlobalIDString = 03C818302AC79FCD0084CC29;
 			remoteInfo = ImageClassification;
 		};
+		84EF1FE92C7850B6005922B4 /* PBXContainerItemProxy */ = {
```

Comment:
All the change here in xcodeproj is to make the `ExecuTorchDemo` app host the test suite `MobileNetClassifierTest` so that the suite can be run on actual AWS devices (coming up next in a separate PR)

Reply:
So, unlike running the test suite in a simulator, running it on actual iOS devices requires a hosting app.  So I have 2 choices:

1. Either create a new empty app to host `MobileNetClassifierTest`
2. or just reuse the ExecuTorchDemo app (as show in my xcode screenshot).  This kind of makes more sense to me because we will also be able to test if the app can be run at all on iOS.  So, two birds one stone.

![Screenshot 2024-08-26 at 11 37 27](https://github.com/user-attachments/assets/04ec0b07-652c-44d7-a805-e3e0fc6ebbfc)


@shoumikhin what would be your preferred choice here from the expert?

Reply:
Btw, the change I'm highlighting in the screenshot is responsible for those magic numbers that you notice (auto-generated by xcode)

Reply:
For this PR to demonstrate using the certs in the workflow I think option 2 makes sense. I'm chatting with @shoumikhin regarding a generic app for benchmarking we will create a new app from scratch with no UI but just tests. @huydhn @shoumikhin once we have that app, can we reuse the provisioning profile and secret directly, or need to re-gen for the new app?

Reply:
From what I see, if we can keep the bundle id `org.pytorch.executorch.demo.test`, then we can reuse the provisioning profile.  Otherwise, we need a new one and store it under a different secret.